src/Explorer/Panes/Tables/AddTableEntityPanel.tsx

```diff
@@ -254,11 +254,14 @@ export const AddTableEntityPanel: FunctionComponent<AddTableEntityPanelProps> =
         {entities.map((entity, index) => {
           return (
             <TableEntity
+              indexofelement={index}
               key={"" + entity.id + index}
               isDeleteOptionVisible={entity.isDeleteOptionVisible}
-              entityTypeLabel={index === 0 && dataTypeLabel}
+              entityTypeLabel={dataTypeLabel}
               entityPropertyLabel={index === 0 && attributeNameLabel}
               entityValueLabel={index === 0 && attributeValueLabel}
+              entityValueAriaLabel={attributeValueLabel}
```

Comment:
The `attributeValueLabel` and `attributeNameLabel` are constants from `EntityTableHelper.tsx`. Why do you need to pass them via parameters? You can just import them from `EntityTableHelper.tsx`?